### PR TITLE
At/cb 97 adjust the operation for the burger menu

### PR DESCRIPTION
**Pull request by Angelbt12**
Implementation of the drop-down menu to show the options in the mobile format
image

### CODE REVIEW

**Review by novadeveloper14 — CHANGES_REQUESTED** [review]
conflicts need to be resolved

**Comment by novadeveloper14** [comment]
The review and comments were left in the following PR which was closed

https://github.com/selsa-inube/crediboard/pull/73

File: src/components/modals/MobileMenu/stories/MobileMenu.stories.tsx
Comment (novadeveloper14) [inline src/components/modals/MobileMenu/stories/MobileMenu.stories.tsx:1]: You forgot to create the props file, which is where the component's props are described and then passed to the story's argTypes. You can use src\components\modals\TextAreaModal\stories\props.ts as a guide. To test the correct functionality of the story, you can run "npm run storybook" in the console.

File: src/components/modals/MobileMenu/styles.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  width: 320px; 
+`;
+
+
+
+export const StyledMenuItem = styled.div`
+  padding: 12px 0; 
+  cursor: pointer;
+  font-size: 14px;
```
Comment (novadeveloper14) [inline src/components/modals/MobileMenu/styles.ts:25]: We should not modify text styles.

**Review by novadeveloper14 — CHANGES_REQUESTED** [review]

**Comment by novadeveloper14** [comment]
fix conflicts

File: src/components/layout/ContainerSections/index.tsx [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import { useState } from "react";
 import { useNavigate } from "react-router-dom";
 import { MdArrowBack, MdMenu } from "react-icons/md";
-import { Button, Text, inube } from "@inube/design-system";
 import { Icon } from "@inubekit/icon";
 import { Stack } from "@inubekit/stack";
 
+import { Button, Text, inube } from "@inube/design-system";
```
[the remark this answers is not in the document]
Comment (Angelbt12) [reply src/components/layout/ContainerSections/index.tsx:7]: ok

File: src/components/layout/ContainerSections/index.tsx [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
 import { TextAreaModal } from "@components/modals/TextAreaModal";
+import { MobileMenu } from "@src/components/modals/MobileMenu/MobileMenu";
```
[the remark this answers is not in the document]
Comment (Angelbt12) [reply src/components/layout/ContainerSections/index.tsx:9]: ok

File: src/components/modals/MobileMenu/stories/MobileMenu.stories.tsx
Comment (Angelbt12) [reply src/components/modals/MobileMenu/stories/MobileMenu.stories.tsx:1]: OK

File: src/components/modals/MobileMenu/stories/props.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
+// src/components/modals/MobileMenu/stories/props.ts
```
Comment (novadeveloper14) [inline src/components/modals/MobileMenu/stories/props.ts:1]: Do not leave comments in the code

File: src/components/modals/MobileMenu/MobileMenu.tsx
Comment (novadeveloper14) [inline src/components/modals/MobileMenu/MobileMenu.tsx:1]: This file should be called index.tsx in any case in any component it should be called index.tsx and styles.ts what distinguishes the file is the name of its folder

**Review by novadeveloper14 — CHANGES_REQUESTED** [review]

File: src/components/modals/MobileMenu/MobileMenu.tsx
Comment (Angelbt12) [reply src/components/modals/MobileMenu/MobileMenu.tsx:1]: ok

File: src/components/modals/MobileMenu/stories/props.ts (hunk 4 above)
Comment (Angelbt12) [reply src/components/modals/MobileMenu/stories/props.ts:1]: ok

**Pull request closed without merging**